Wikipedia/UI-V5/WMFArticleViewController.m

```diff
@@ -491,7 +541,9 @@ - (WMFArticleReadMoreCell*)readMoreCellAtIndexPath:(NSIndexPath*)indexPath {
 
 - (CGFloat)tableView:(UITableView*)tableView heightForHeaderInSection:(NSInteger)section {
     if ([self isLeadSection:section]
-        || ([self isTOCSection:section] && section != self.indexSetOfTOCSections.firstIndex)) {
+        || ([self isTOCSection:section] && section != self.indexSetOfTOCSections.firstIndex)
+        || ![self tableView:tableView viewForHeaderInSection:section]
```

Comment:
i'm a bit baffled as to how this doesn't cause an infinite loop... don't you need to know the header height to get the header view?